imports/ui/components/auth/Register.jsx

```diff
@@ -48,35 +48,41 @@ export default class Register extends Component {
         let currency = {symbol: "$", name: "Dollar", symbol_native: "$", decimal_digits: 2, rounding: 0},
         emailNotification = true;
 
-        Accounts.createUser({
-            [key]: selector[key],
-            password,
-            profile: {fullName, currency, emailNotification }
-        }, (err) => {
-            if(err){
-                this.props.showSnackbar({
-                    activeSnackbar: true,
-                    barMessage: err.reason,
-                    barIcon: 'error_outline',
-                    barType: 'cancel'
-                });
-            }else{
-                this.props.showSnackbar({
-                    activeSnackbar: true,
-                    barMessage: 'Successfully Registered',
-                    barIcon: 'done',
-                    barType: 'accept'
+        var account = {account: {password, fullName, selector}};
+        Meteor.call("registerUser", account, (err) => {
```

Comment:
Sorry I forgot to close this pull request, new pull request is https://github.com/AppBakerZ/FinanceBakerZ/pull/73.